# file: C:\Users\erena\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\pandas\core\array_algos\take.py
# hypothesis_version: 6.140.2

['F', 'M8[ns]', 'bool', 'datetime64[ns]', 'float32', 'float64', 'int16', 'int32', 'int64', 'int8', 'm', 'm8[ns]', 'mM', 'object', 'timedelta64[ns]']